packages/core/src/components/TextField/__snapshots__/TextField.test.jsx.snap

```diff
@@ -0,0 +1,30 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`TextField masked renders currency mask 1`] = `
+<div
+  className="ds-u-clearfix"
+>
+  <FormLabel
+    component="label"
+    fieldId="textfield_27"
+  >
+    Foo
+  </FormLabel>
+  <div
+    className="ds-c-field-mask ds-c-field-mask--currency"
+  >
+    <div
+      className="ds-c-field__before ds-c-field__before--currency"
+    >
+      $
+    </div>
+    <input
+      aria-label="Enter amount in dollars"
```

Comment:
@sawyerh I'd like this `aria-label` to include the printed label + more descriptive information. Screenreader users who are moving around via the Tab key might settle on the input, and in doing so, miss the `<label>` being read out. So all they'll hear is "Enter amount in dollars."  but it's not clear what they should enter dollars for.

Reply:
PTAL